examples/bin/run-druid

```diff
@@ -34,7 +34,7 @@ else
   CONFDIR="$2"
 fi
 
-CONFDIR="$(cd "$CONFDIR" && pwd)/druid"
+CONFDIR="$(cd "$CONFDIR" && pwd)"
```

Comment:
Why was the /druid removed?  It's a breaking change for older version users.  It doesn't seem like an absolutely necessary change.  This file is one of the bin scripts in the apache-druid.tar.gz distribution although the directory name is "examples". Please avoid similar changes in the future release.  Thanks.

Reply:
Hi @knoguchi,

It was changed to comport with the new directory structure in this patch, where we ship with multiple potential configurations for various sizes of servers. (It no longer makes sense to add `/druid` to the conf dir, since we provide the full path to a Druid-specific conf dir in the supervise configs.)

I think we should have noted this in the release notes, though.